front-end/src/components/CreateTodo.tsx

```diff
@@ -0,0 +1,23 @@
+import React, {} from "react";
+import { TodoType } from "../types";
+import useCreateTodo from "../hooks/useCreateTodo";
+
+function CreateTodo() {
+    const AddHandler = useCreateTodo()
+    const todo_item: TodoType = {
+        id: 3,
+        title: "test",
+        description: "test",
+        needsReminder: false,
+        createdAt: Date(),
+        createdBy: "test",
+        };
+    
+    return (
+        <button onClick={() => {
+            AddHandler(todo_item);
```

Comment:
Should be `onClick={handleAddTodo}` where `handleAddTodo` will be returned by hook.